pinot-common/src/main/java/com/linkedin/pinot/common/data/Schema.java

```diff
@@ -121,6 +130,19 @@ public void setMetricFieldSpecs(@Nonnull List<MetricFieldSpec> metricFieldSpecs)
     }
   }
 
+  @Nonnull
+  public List<DateTimeFieldSpec> getDateTimeFieldSpecs() {
+    return _dateTimeFieldSpecs;
+  }
+
+  public void setDateTimeFieldSpecs(@Nonnull List<DateTimeFieldSpec> dateTimeFieldSpecs) {
+    Preconditions.checkState(_dateTimeFieldSpecs.isEmpty());
+
+    for (DateTimeFieldSpec dateTimeFieldSpec : dateTimeFieldSpecs) {
+      addField(dateTimeFieldSpec);
```

Comment:
Do you need to add it to the set as well as to the list?

Reply:
I tried to make it consistent with the already existing metricFieldSpecs and dimensionFieldSpecs. Those have sets and lists, and methods to add into both. Not sure why it was done for those, and not sure if they will be used for dateTimeFieldSpec, but wanted to have them there to be consistent with how it is designed.